skultrafast/messpy.py

```diff
@@ -77,14 +81,16 @@ def process_vis(self,
         pol = pol.cut_freqs([vis_range], invert_sel=True)
         return pol.para, pol.perp, pol
 
-    def process_ir(self, t0=0, max_scans=None):
+    def process_ir(self, t0=0, max_scans=None, invert_data= False):
```

Comment:
This makes sense at least right now, still it may be better to allow setting the signal sign in messpy itself.